.github/workflows/iroha2-dev-pr-title-commit-msg.yml

```diff
@@ -1,17 +1,17 @@
-name: I2::Dev::Title
+name: I2::Dev::Titles
 
 on:
   pull_request:
     branches: [iroha2-dev]
-    types: [opened, edited, reopened]
+    types: [opened, edited, reopened, synchronize]
 
 jobs:
-  check:
+  pr-title-check:
     runs-on: ubuntu-latest
     # TODO: Add labels when https://github.com/actions/first-interaction/issues/10 is fixed.
     steps:
       - name: Feature
-        uses: actions-ecosystem/action-regex-match@v2
+        uses: actions-ecosystem/action-regex-match@v2.0.2
```

Comment:
Please revert. These actions might be pushing minor fixes that we'd have to manually track through. We don't want to do it manually.